x2gbfs/providers/cantamen.py

```diff
@@ -238,7 +238,10 @@ def _extract_form_factor(self, bookee: Dict) -> str:
         return 'cargo_bicycle' if bookee['Class'] == 'bike' else 'car'
 
     def _as_vehicle_type_id(self, vehicle_name: str) -> str:
-        return vehicle_name.lower().translate({ord(c): None for c in ',< ().äöüßé/+-'})
+        # returns the vehicle name as lowercased string, filtered to characters a-z only.
+        # (Restrictive filtering is required as consuming systems like lamassu do only support a subset
+        # of characters allowed by the GBFS spec.)
+        return re.sub('[^a-z]+', '', vehicle_name.lower())
```

Comment:
shall we include numbers to distinguish, e.g, BMW 1er  / BMW 3er

Reply:
Indeed. Added 0-9 as well.